dataclients/kubernetes/kube.go

```diff
@@ -188,9 +190,11 @@ type Client struct {
 	reverseSourcePredicate   bool
 	pathMode                 PathMode
 	quit                     chan struct{}
-	namespace                string
```

Comment:
If we drop namespace, do we break any filter by namespace or so?
I don’t remember if we have such a feature and can not check it right now, but maybe something to check.

Reply:
I don't think so, because then compilation would fail. It's a private field as well so I don't think it's used by any filters.

Reply:
as far as I can tell from the changes, the namespace handling is good.

Reply:
thanks